cmake/jakt-executable.cmake

```diff
@@ -35,7 +35,7 @@ function(add_jakt_compiler_flags target)
   )
   if (MSVC)
     # For clang-cl, which shows up to CMake as MSVC and accepts both kinds of arguments
-    target_compile_options("${target}" PRIVATE /permissive- /utf-8 /EHsc- -W4)
+    target_compile_options("${target}" PRIVATE /permissive- /utf-8 /EHsc- -W[123])
```

Comment:
If this truly is the only way, then it should be fixup squashed back into the first commit.

However, when I tested it, using `/W4` rather than `-W4` compiled just fine.

Reply:
Just double checked as I don't think I'd tried `/W4`, but I get the same stack of errors no matter if it's `-W4` or `/W4`. Tons of `error-unused`, `parenthesis-inequality`, things that have explicitly been shut off by other options. As I linked up top, apparently this is expected behavior for certain logging levels in MSVC.